src/CLI/actioner/sonic-cli-mac.py

```diff
@@ -42,20 +41,10 @@ def call_method(name, args):
     method = plugins[name]
```

Comment:
You dont need the plugins, register() and call_method()

Reply:
Done